src/components/Modal/Modal.vue

```diff
@@ -13,7 +13,10 @@
         aria-labelledby="header"
         aria-describedby="modalDescription"
         :style="{'width': width}"
-        class="relative bg-white flex flex-col overflow-y-auto shadow rounded-lg p-5 max-h-5/6"
+        :class="[
+          'relative bg-white flex flex-col overflow-y-auto shadow rounded-lg p-5 max-h-5/6',
+          {'!p-0' : noPadding}
```

Comment:
```suggestion
          :class="['relative bg-white flex flex-col overflow-y-auto shadow rounded-lg max-h-5/6', noPadding ? 'p-0' : 'p-5']"
```

Reply:
so you make sure you use one **or** the other, and don't worry about one overwriting the other or viceversa

Reply:
This works, but it's an anti-pattern to how we've always done things in this repo - i'm not sure it's as easy to read, @juanfriss

Reply:
I wouldn't say it is an "anti-pattern", as using a ternary expression is suggested in the Vue docs themselves [here](https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes).

If with "anti-pattern" you meant "different to what we use to do", then I agree. However, I don't like what we were doing in the repo (using `p-5` and then using `!` so we overwrite the previous class)

Reply:
I feel `noPadding ? 'p-0' : 'p-5'` reads like 'if no padding then use class p-0, else use class p-5'.

Reply:
I updated it!